dask_bigquery/tests/test_core.py

```diff
@@ -3,15 +3,20 @@
 import uuid
 from datetime import datetime, timedelta, timezone
 
+import dask.dataframe as dd
 import google.auth
 import pandas as pd
+import pyarrow as pa
 import pytest
 from dask.dataframe.utils import assert_eq
+from distributed.utils_test import cleanup  # noqa: F401
+from distributed.utils_test import client  # noqa: F401
 from distributed.utils_test import cluster_fixture  # noqa: F401
-from distributed.utils_test import client, loop  # noqa: F401
+from distributed.utils_test import loop  # noqa: F401
+from distributed.utils_test import loop_in_thread  # noqa: F401
```

Comment:
I don't think you're actually using the additional imports, are you?

FWIW it's also possible to get all fixtures by setting

`from distributed.utils_test import * # noqa: F401`

which is a bit cleaner

Reply:
I ended up not doing `from distributed.utils_test import *`, because that also imports `pytest` from there.

Reply:
I tried to limit the list of fixtures to ones referenced in the test, but `pytest` is unhappy without the other fixtures. Added them back.